ios/StoresTests/StoreTests.swift

```diff
@@ -59,17 +59,6 @@ class StoreTests: XCTestCase {
 
         XCTAssertEqual(obj, stores.get(.codableKey))
     }
-    
-    func testUserDefaultArrayBasedKey() {
-        let expected = [
-            ExampleStruct(name: "User 1", age: 11),
-            ExampleStruct(name: "User 2", age: 22)
-        ]
-
-        stores.set(.structArrayKey, value: expected)
-        
-        XCTAssertEqual(expected, stores.get(.arrayBasedKey))
-    }
```

Comment:
We don't still want the test?

Reply:
It's a bad test. Introduced here 0c8ecc37d99039451fa40ec5ccea36e02f4068a2